src/components/Calculator.js

```diff
@@ -24,126 +24,117 @@ Output.propTypes = {
   result: PropTypes.string,
 };
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.handleClick = this.handleClick.bind(this);
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
-  }
+function Calculator() {
```

Comment:
[Optional] I suggest you use ES6 syntax (arrow function)as it's a shorter syntax and thus requires less code.

Reply:
You're right. Thank you sir!